actix-threadpool/Cargo.toml

```
@@ -19,7 +19,7 @@ path = "src/lib.rs"
 
 [dependencies]
 derive_more = "0.15"
-futures = "0.1.25"
+futures =  { package = "futures-preview", version  = "0.3.0-alpha.18" }
```

```suggestion
futures-preview = "0.3.0-alpha.19"
```

IMO: `futures-preview` is more common?

---

So, we should use `futures 0.3.0` now

---

0.3.1 actually is out for a hotfix